weather@mockturtl/README.md

```diff
@@ -33,25 +33,25 @@ You can also save locations what you entered manually and switch between them in
 
 ### OpenWeatherMap
 
-Worldwide Online Weather service by OpenWeather Ltd founded in 2012 with headquaters in London UK. [Link](https://openweathermap.org/) 
+Worldwide Online Weather service by OpenWeather Ltd founded in 2012 with headquaters in London UK. [Link OpenWeatherMap Website](https://openweathermap.org/) 
```

Comment:
"Link" is not needed in any of the link texts, just remove them and leave them like this "{Provider} Website"